pkg/component/garden/system/virtual/virtual.go

```diff
@@ -145,16 +145,20 @@ func (g *gardenSystem) computeResourcesData() (map[string][]byte, error) {
 				Labels: map[string]string{v1beta1constants.GardenRole: "admin"},
 			},
 			Rules: []rbacv1.PolicyRule{
+				{
+					APIGroups: []string{gardencorev1beta1.GroupName},
+					Resources: []string{"*"},
+					Verbs:     []string{"create", "delete", "deletecollection", "get", "list", "watch", "patch", "update", "manage-members", "modify-spec-tolerations-whitelist", "modify-spec-kubernetes", "modify-spec-machineimages", "modify-spec-providerconfig", "mark-autonomous"},
```

Comment:
Please update the list [here](https://github.com/gardener/gardener/blob/11a5f6bb81962c63c8467f71e70dbceef0602175/charts/gardener/controlplane/charts/application/templates/rbac-user.yaml#L18-L31) also, or do we not need it since the charts are deprecated?